From 4c219798736889b5bbb4f84ac5da71283c8536d2 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 19 Feb 2016 16:19:15 +0000
Subject: [PATCH 0735/1077] MANTIS 40061 extension: Extend Conveyor-Component
 for extended ConveyorDev.idl

codes.idl
V1.3.6

added BAGGAGE_TOO_SMALL



git-svn-id: svn://localhost/SelfServiceCommon/trunk@990 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index e992b5a1..50a2565f 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -47,6 +47,11 @@ Changed:   Added baggage handling status codes (Nicolas Bruns, 29.06.2004)
 @li added DS_TYPES_SSML10; DS_TYPES_SSML11
 - clarified: use DS_TYPES_SSML11= 25010 instead of DS_TYPES_SSML11= 25001
                
+@date 19.02.2016
+@li V1.3.6
+@li Mantis 0040061: Extend Conveyor-Component for extended ConveyorDev.idl
+@li added BAGGAGE_TOO_SMALL
+               
 */
 
 #ifndef CODES_IDL
@@ -312,6 +317,7 @@ module statuscodes
     const long BAGGAGE_TUB_REQUIRED          = 152; /**< <b>Scope:</b> private + platform; materna ccm extension */
     const long BAGGAGE_HANDLE_EXTENDED       = 153; /**< <b>Scope:</b> private + platform; materna ccm extension */
     const long BAGGAGE_TOO_WIDE              = 154; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_TOO_SMALL             = 155; /**< <b>Scope:</b> private + platform; materna ccm extension */
 
     const long FORMAT_ERROR                = 201; /**< <b>Scope:</b> private + platform (platform only for output classes) */
     const long LENGTH_ERROR                = 202; /**< <b>Scope:</b> private + platform (platform only for output classes) */
-- 
2.41.0.windows.1

